assets/config.php

```diff
@@ -178,6 +178,11 @@
         '--realpath' => true,
     ],
 
+    /**
+     * The class and method name used by tenants:migrate command with --only-selected option.
+     */
+    'migration_filter_tenants_method' => [\App\Repositories\TenantRepository::class, 'filterBy'],
```

Comment:
This puts a burden on the developer by having to create this class.

Reply:
It's optional actually.